docs/current_docs/cookbook/cookbook.mdx

```diff
@@ -519,26 +519,59 @@ dagger call build \
   --dockerfile https://github.com/vimagick/dockerfiles#master:registry-cli/Dockerfile
 ```
 
+### Add OCI annotations to image
+
+The following Dagger Function adds OpenContainer Initiative (OCI) annotations to an image.
```

Comment:
I would potentially add a link to https://github.com/opencontainers/image-spec/blob/main/annotations.md here.

And maybe https://github.com/opencontainers/image-spec/blob/main/config.md in the labels section.